src/plugins/timelist/Timelist.vue

```diff
@@ -163,6 +180,57 @@ export default {
                 this.listActivities();
             }
         },
+        addItem(domainObject) {
+            this.planObjects = [domainObject];
+            this.resetPlanData();
+            if (domainObject.type === 'plan') {
+                this.getPlanDataAndSetConfig({
+                    ...this.domainObject,
+                    selectFile: domainObject.selectFile
+                });
+            }
+        },
+        addToComposition(telemetryObject) {
+            if (this.planObjects.length > 0) {
+                this.confirmRemoval(telemetryObject);
```

Comment:
can we name this something like `confirmReplaceCurrent` or something? it was a little confusing to me initially.

Reply:
Using confirmReplacePlan instead

Reply:
Much better than mine!